watchman/checks.py

```diff
@@ -9,6 +9,8 @@
 from django.core.mail import EmailMessage
 from django.db import connections
 
+import pika
```

Comment:
My main concern is the additional requirement of another library here - especially for users which aren't using rabbitmq, or are using another library.

I have two ideas that I think I would be ok with:

* Make this check optional, including this import (eg. `try ... except ImportError` and handle things smartly)
* Split this check out into a separate package (`django-watchman-amqp`?) which makes it easy to install on demand for apps which need it

@pobed2 what do you think?